configuration/kilifi_pipeline_config.json

```
@@ -0,0 +1,72 @@
+
+    {"RapidProKey": "Rqa_S01E02 (Text) - gpsdd_kilifi_s01e02_activation", "PipelineKey": "kilifi_rqa_s01e02_raw", "IsActivationMessage": true},
+    {"RapidProKey": "Rqa_S01E02 (Run ID) - gpsdd_kilifi_s01e02_activation", "PipelineKey": "kilifi_rqa_s01e02_run_id"},
+    {"RapidProKey": "Rqa_S01E02 (Time) - gpsdd_kilifi_s01e02_activation", "PipelineKey": "sent_on"},
+
+    {"RapidProKey": "Rqa_S01E03 (Text) - gpsdd_kilifi_s01e03_activation", "PipelineKey": "kilifi_rqa_s01e03_raw", "IsActivationMessage": true},
+    {"RapidProKey": "Rqa_S01E03 (Run ID) - gpsdd_kilifi_s01e03_activation", "PipelineKey": "kilifi_rqa_s01e03_run_id"},
+    {"RapidProKey": "Rqa_S01E03 (Time) - gpsdd_kilifi_s01e03_activation", "PipelineKey": "sent_on"},
+
+    {"RapidProKey": "Rqa_S01E04 (Text) - gpsdd_kilifi_s01e04_activation", "PipelineKey": "kilifi_rqa_s01e04_raw", "IsActivationMessage": true},
+    {"RapidProKey": "Rqa_S01E04 (Run ID) - gpsdd_kilifi_s01e04_activation", "PipelineKey": "kilifi_rqa_s01e04_run_id"},
+    {"RapidProKey": "Rqa_S01E04 (Time) - gpsdd_kilifi_s01e04_activation", "PipelineKey": "sent_on"},
+
+    {"RapidProKey": "Rqa_S01E05 (Text) - gpsdd_kilifi_s01e05_activation", "PipelineKey": "kilifi_rqa_s01e05_raw", "IsActivationMessage": true},
+    {"RapidProKey": "Rqa_S01E05 (Run ID) - gpsdd_kilifi_s01e05_activation", "PipelineKey": "kilifi_rqa_s01e05_run_id"},
+    {"RapidProKey": "Rqa_S01E05 (Time) - gpsdd_kilifi_s01e05_activation", "PipelineKey": "sent_on"},
+
+    {"RapidProKey": "Constituency (Text) - gpsdd_kilifi_s01_demog", "PipelineKey": "location_raw"},
+    {"RapidProKey": "Constituency (Time) - gpsdd_kilifi_s01_demog", "PipelineKey": "location_time"},
+    {"RapidProKey": "Gender (Text) - gpsdd_kilifi_s01_demog", "PipelineKey": "gender_raw"},
+    {"RapidProKey": "Gender (Time) - gpsdd_kilifi_s01_demog", "PipelineKey": "gender_time"},
+    {"RapidProKey": "Age (Text) - gpsdd_kilifi_s01_demog", "PipelineKey": "age_raw"},
+    {"RapidProKey": "Age (Time) - gpsdd_kilifi_s01_demog", "PipelineKey": "age_time"}
+  ],
+  "ProjectStartDate": "2021-03-01T10:30:00+03:00",
+  "ProjectEndDate": "2100-01-01T00:00:00+03:00",
+  "FilterTestMessages": true,
+  "MoveWSMessages": true,
+  "AutomatedAnalysis": {
+    "GenerateCountyThemeDistributionMaps": true,
+    "GenerateConstituencyThemeDistributionMaps": true
+  },
+  "DriveUpload": {
+    "DriveCredentialsFileURL": "gs://avf-credentials/pipeline-runner-service-acct-avf-data-core-64cc71459fe7.json",
+    "ProductionUploadPath": "gpsdd_analysis_outputs/kilifi/production.csv",
+    "MessagesUploadPath": "gpsdd_analysis_outputs/kilifi/messages.csv",
+    "IndividualsUploadPath": "gpsdd_analysis_outputs/kilifi/individuals.csv",
+    "AutomatedAnalysisDir": "gpsdd_analysis_outputs/kilifi/automated_analysis"
+  },
+  "MemoryProfileUploadBucket": "gs://avf-pipeline-logs-performance-nearline",
+  "DataArchiveUploadBucket": "gs://pipeline-execution-backup-archive",
+  "BucketDirPath": "2021/GPSDD"
```

```suggestion
  "BucketDirPath": "2021/GPSDD/KILIFI"
```
I think this should be separate for each pipeline, so that we get daily archives for each location when the pipelines are running in parallel

---

updated BucketDirPath to _"2021/GPSDD/{LOCATION}"_. Thanks